recipe/meta.yaml

```diff
@@ -170,6 +167,8 @@ outputs:
         - pytorch
         - pyyaml
         - requests
+        - rucio-clients <=1.29.12
+        - ruff ==0.0.275
```

Comment:
We've been using 0.0.278 in pre-commit but I see 0.0.276 is the only one on conda-forge at the moment. Don't suppose it matters much.

Reply:
Changed to 0.0.278.